src/migrations/migration_runner.js

```diff
@@ -114,9 +129,20 @@ export default class MigrationRunner {
     while (totalCount !== 0) {
       info = `Iteration: ${iteration}\n`;
       upgradedThisIteration = 0;
-      for (const migration of migrations) {
+      for (let i = 0; i < migrations.length; i++) {
+        const migration = migrations[i];
+        if (this._debug) {
+          this._logger.debug('--------------------------------------------------------------------------------');
+          this._logger.debug(`Migration ${i + 1} out of ${migrationsNo}: ${migration.constructor.description}`);
+        }
         try {
+          const start = Date.now();
           const count = await migration.upgrade();
+          if (this._debug) {
+            const stop = Date.now();
+            this._logger.debug(`${count} upgraded objects`)
+            this._logger.debug(`${((stop - start)/1000).toFixed(2)} s execution time`);
```

Comment:
```suggestion
            this._logger.debug(`${((stop - start)/1000).toFixed(2)}s execution time`);
```
`7.25 s execution time` -> `7.25s execution time`?